SequenceAnalysis/build.gradle

```diff
@@ -67,3 +67,19 @@ if (project.findProject(BuildUtils.getTestProjectPath(project.gradle)) != null &
       dependsOn(createPipelineConfigTask)
    }
 }
+
+project.task("copyJars",
+        type: Copy,
+        group: "Build",
+        description: "Copy commons-math3 JAR to module's lib directory",
+        { CopySpec copy ->
+           copy.setDuplicatesStrategy(DuplicatesStrategy.EXCLUDE)
+           copy.from(project.configurations.external)
+           copy.into new File("${project.labkey.explodedModuleLibDir}")
+           copy.include {
+              "**commons-math3-**.jar"
+           }
+        }
+)
+
+project.tasks.processModuleResources.dependsOn(project.tasks.copyJars)
```

Comment:
This is probably going to happen too early since `processModuleResources` can run well before `populateExplodedLib` and the latter task starts out by removing the `explodedModule/lib` directory.  I think you probably want:
```
project.tasks.module.dependsOn(copyJars)
copyJars.mustRunAfter(project.tasks.populateExplodedLib)
```

Reply:
ok. this seems to be working locally (including after clean), but i see your point